docs/en/engines/table-engines/mergetree-family/invertedindexes.md

```diff
@@ -448,9 +459,12 @@ Direct read is controlled by two settings:
 Also, the text index must be fully materialized to use direct reading (use `ALTER TABLE ... MATERIALIZE INDEX` for that).
 
 **Supported functions**
+
 The direct read optimization supports functions `hasToken`, `hasAllTokens`, and `hasAnyTokens`.
 These functions can also be combined by AND, OR, and NOT operators.
-The WHERE clause can also contain additional non-text-search-functions filters (for text columns or other columns) - in that case, the direct read optimization will still be used but less effective (it only applies to the supported text search functions).
+The `WHERE` clause can also contain additional non-text-search-functions filters (for text columns or other columns) - in that case, the direct read optimization will still be used but less effective (it only applies to the supported text search functions).
+
+If text index is created with an `array` tokenizer, the direct read is also supported for functions `equals`, `has`, `mapContainsKey`, and `mapContainsValue`.
```

Comment:
I think we better move this sentence up directly after l. 463 (`The direct read optimization supports functions [...]`).